content/codespaces/setting-up-your-codespace/configuring-codespaces-for-your-project.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Configuring Codespaces for your project
-intro: You can set up a default configuration for every new codespace for your repository to ensure that contributors have all the tools and settings they need in their online development environment.
+intro: You can use a `devcontainer.json` file to define a {% data variables.product.prodname_codespaces %} environment for your repository. 
 product: '{% data reusables.gated-features.codespaces %}'
```

Comment:
for ( index = 0 to input.length-1 ) {

				bool sorted = true

				for ( j = 0 to j < input.length-1-index ) {

						if ( input[ j ] > input[ j+1 ] ) {

								swap input[ j+1 ] & input[ j ]

								sorted = false

						}

				}

				// Stop, once the array is sorted
				if ( sorted )
						return;
		}